core/src/block.rs

```diff
@@ -397,11 +415,6 @@ impl ValidBlock {
             signatures,
         } = self;
 
-        #[allow(clippy::expect_used)]
-        let signatures: SignaturesOf<ValidBlock> = signatures
-            .try_into()
-            .expect("Expected at least one signature");
-
         CommittedBlock {
             event_recommendations,
             header,
```

Comment:
Can we replace this with `..self` instead?

Reply:
Sadly same types are required for this.